css/css-values/percentage-rem-low.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>REM value with :root set to low percentage</title>
+<link rel="author" title="Jon Ege Ronnenberg" href="mailto:jon.ronnenberg+wpt@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-values/#rem">
+<link rel="help" href="https://drafts.csswg.org/css-fonts/#length-percentage-size-value">
+<link rel="match" href="../reference/ref-filled-green-100px-square-only.html">
+<meta name="assert" content="When :root font-size is 25%, a 25rem length must be 100px, if the browser default for :root font-size is 16px.">
+<style>
+  :root {
+     /* default is 16px in all desktop browsers */
+     /* the math for 100px is: 16px * 0.25 = 4px/rem -> 100px / 4px/rem = 25rem = 100px */
+    font-size: 25%;
+  }
+  .rem {
+    box-sizing: border-box;
```

Comment:
Is this needed to expose the bug, given that there are no borders? This test is already clean and you'll find many that are very messy, but if https://web-platform-tests.org/writing-tests/general-guidelines.html#be-minimal can be taken a step further, why not? :)

Reply:
You're right. `box-sizing: border-box;` is not needed. I'll remove it.